join_skip.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Join SKIP
+layout: post
+---
+
+## Join Us: Become a SKIP Member!
+
+Are you eager to be part of a dynamic and impactful student association? At SKIP, we welcome students who are enthusiastic about enhancing the academic and social experience of their peers. Whether you're interested in taking a leadership role as a board member or providing valuable support as a regular member, there's a place for you in our community.
+
+1. **Regular Members:**
+As a regular member, you can actively contribute to the success of SKIP. Your participation in events and feedback are highly valued. Additionally, you'll enjoy perks like priority access to events and discounts. If you're ready to dive in and be an essential part of the SKIP experience, apply today by clicking
```

Comment:
Looks good! Just the link is missing.

Reply:
Wanna leave this PR open until a link exists?